test.sh

```diff
@@ -40,3 +40,7 @@ for d in $PKG; do
   fi
 done
 
+go vet -all ./...
+golangci-lint run -D errcheck ./...  # TODO: Enable errcheck back.
```

Comment:
I assume we have places in the pprof CLI that we don't check errors so we need to disable this in the linter? Are you  going to fix it separately?

Reply:
Yes, the linter complained about a few places there. I don't plan to work on this soon as it's not a regression.